test/common/config/BUILD

```diff
@@ -276,6 +276,17 @@ envoy_cc_test(
     ],
 )
 
+envoy_cc_test(
```

Comment:
Thanks for the `.png`; can you put it in `source/docs`?

Reply:
Done.